lib/yearRange.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+/* eslint-disable */
+
+/**
+ * Function to generate a span of years based on minimum and maximum parameters
+ *
+ * @param {Object} input - inputs in this plugin's instance and their values as {String}
+ * @param {Object} minYear - inputs minimum year value {String}
+ * @param {Object} maxYear - inputs maxmium year value {String}
+ * *
+ * @module yearRange
+ */
+
+module.exports = function yearRange(input, minYear, maxYear) {
+  var select = input.querySelector('.year-range');
+
+  for (var i = minYear; i <= maxYear; i++) {
```

Comment:
It looks like options are being generated entirely client side. We need these to be generated in HTML, not through JS